discovery-infra/test_infra/helper_classes/kube_helpers/installenv.py

```diff
@@ -0,0 +1,232 @@
+import waiting
+import yaml
+
+from typing import Optional, Union, Dict
+from pprint import pformat
+
+from kubernetes.client import ApiClient, CustomObjectsApi
+from kubernetes.client.rest import ApiException
+
+from tests.conftest import env_variables
+
+from .globals import *
+from .base_resource import BaseCustomResource
+from .secret import deploy_default_secret, Secret
+from .common import logger
+
+
+class InstallEnv(BaseCustomResource):
+    """
+    InstallEnv is used to generate cluster iso.
+    Image is automatically generated on CRD deployment, after InstallEnv is
+    reconciled. Image download url will be exposed in the status.
+    """
+    _api_group = 'adi.io.my.domain'
```

Comment:
Let's be consistent with ``_api_group`` as in ``ClusterDeployment``

Reply:
cluster deployment CRD is part of hive api group because this CRD is belong to hive
installenv is our CRD so it won't work if we deploy it to hive api group

Reply:
I mean the variable name, not the value